test/integration/data/sshkey/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
makes sense,  I can generate them to a tmp directory and remove them when complete.